components/navbar/MenuManager.js

```diff
@@ -0,0 +1,37 @@
+import { Skeleton } from "@mui/material";
+import DesktopMenu from "./DesktopMenu";
+import MobileMenu from "./MobileMenu";
+
+export default function MenuManager({ version, loggedIn, pages }) {
```

Comment:
It is better to do the export default at the end of file?

Reply:
Done